src/WebJobs.Script.WebHost/Diagnostics/FastLogger.cs

```diff
@@ -28,12 +28,18 @@ public FastLogger(string hostName, string accountConnectionString)
 
         public async Task AddAsync(FunctionInstanceLogEntry item, CancellationToken cancellationToken = default(CancellationToken))
         {
-            // Convert Host to Protocol so we can log it 
-            var settings = new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore };
-            var jsonClone = JsonConvert.SerializeObject(item, settings);
-            var item2 = JsonConvert.DeserializeObject<FunctionInstanceLogItem>(jsonClone);
-            item2.FunctionName = Utility.GetFunctionShortName(item2.FunctionName);
-            await _writer.AddAsync(item2);
+            await _writer.AddAsync(new FunctionInstanceLogItem
```

Comment:
Some background: 
SDK does not take a direct dependency on the fast Logging dll.  Script depends on both, and sets up JobHostConfiguration to use Logging 

That means SDK and logging can’t share an item type declaration. The serialization was a “cheap” way of converting between the two – and also ensuring we didn’t miss any properties.